wearipedia/devices/fitbit/fitbit_charge_4_gen.py

```diff
@@ -207,4 +210,40 @@ def create_syn_data(start_date, end_date):
         full_dict["distance"].append(activity[4])
         full_dict["minutesSedentary"].append(activity[5])
 
+    # encapsulate to match original data shape
```

Comment:
All of the following for-loops can be condensed into a single double for loop: for key in full_dict.keys(), for e in full_dict[key], do X.

Reply:
Actually, all of this code might be unecessary. If the goal is just to have a dictionary of form {key: [{data_type: data}]}, in the "for date in synth_dates:" for loop above you could have just did append({data_type: data}) where data is something like activity[0].